src/main/commandServer/httpCommandServer.ts

```diff
@@ -54,7 +54,8 @@ export class HttpCommandServer {
         diagnostic_ids:        this.diagnosticIDsForCategory,
         diagnostic_checks:      this.diagnosticChecks,
       },
-      PUT: {
+      POST: { diagnostic_checks: this.diagnosticRunChecks },
```

Comment:
That was dumb… fixed `command-api.yaml` to match reality. (It's not a `PUT` because we're not replacing anything with updated data; we're causing the server to run some action.)